pkgs/top-level/all-packages.nix

```
@@ -803,6 +805,10 @@ with pkgs;
 
   sea-orm-cli = callPackage ../development/tools/sea-orm-cli { };
 
+  vcpkg-tool = callPackage ../development/tools/vcpkg-tool { fmt = fmt_10; };
```

Since we are now in the by-name era, can you move this file `development/tools/vcpkg-tool/default.nix` to `by-name/vc/vcpkg-tool/package.nix`?

```suggestion
  vcpkg-tool = callPackage ../by-name/vc/vcpkg-tool/package.nix { 
    fmt = fmt_10; 
  };
```